client/src/components/inventory/HardwareItem.tsx

```diff
@@ -231,7 +231,7 @@ class HardwareItemBase extends React.Component<Item & HardwareItem, HardwareItem
 
         const editBtn = this.props.user && this.props.user.admin ? (<Button size="mini" basic primary={true} icon labelPosition="left">
             <Icon name="pencil"/>
-            <Link to={"/item/" + this.props.id}>Edit</Link>
+            <Link to={"admin/items/" + this.props.id}>Edit</Link>
```

Comment:
btw would prefer template strings ``admin/items/${this.props.id}`` since I think it's a bit safer than string concat this way